src/app/docs/dev-guide/ollama/page.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Configuration Ollama
+nextjs:
+  metadata:
+    title: Configuration Ollama
+    description: 'HackingBuddyGPT: How to set up ollama, the deployment environment?'
+---
+
+system minimum configuration
+
+- RAM 16G
+- hard drive space at least 20G (Depends on the ollama model size you use)
+
+ollama download
+
+(You need to download and run the bad llama!!)
+
+~~~bibtex
+//choose one model 
+https://ollama.com/reefer/erphermesl3
+https://ollama.com/gdisney/zephyr-uncensored
+https://ollama.com/gdisney/mistral-uncensored
+https://ollama.com/gdisney/mixtral-uncensored
+https://ollama.com/gdisney/orca2-uncensored
+https://ollama.com/jimscard/blackhat-hacker
+
+~~~
+
+Depending on your system environment, you can choose to use `cp` to copy the model or modify `.env ` and other files to import ollama
+
+~~~bibtex
+//then copy the selected model into gpt-3.5-turbo
+ ollama cp gdisney/mistral-uncensored gpt-3.5-turbo
```

Comment:
this will break some of the internal logic of hackingBuddyGPT (e.g., token estimation as this uses the given model name to switch between used tokenizers), maybe mention that this is not recommended?